mhctools/epitope_collection.py

```diff
@@ -33,7 +33,7 @@ def __init__(
         Collection.__init__(
             self,
             elements=binding_predictions,
-            path=path,
+            sources=[path] if path else [],
```

Comment:
I think in Varcode it's typical to have a path (and if you manually supply an empty one, so be it) whereas here the default is to not have a path.